content/en/docs/languages/go/quickstart.md

```diff
@@ -21,8 +21,8 @@ spelling: cSpell:ignore Fatalf GOPATH
    1. Install the protocol compiler plugins for Go using the following commands:
 
       ```sh
-      $ go install google.golang.org/protobuf/cmd/protoc-gen-go@v1.28
-      $ go install google.golang.org/grpc/cmd/protoc-gen-go-grpc@v1.2
+      $ go install google.golang.org/protobuf/cmd/protoc-gen-go@v1.34.2
+      $ go install google.golang.org/grpc/cmd/protoc-gen-go-grpc@v1.4.0
```

Comment:
We had a new release of `protoc-gen-go-grpc` yesterday, can you please update the version to 1.5.1?

Reply:
Certainly, I just pushed a new commit.